SMCKit/SMC.swift

```diff
@@ -1167,6 +1370,46 @@ public struct SMC {
     */
     private func callSMC(inout inputStruct : SMCParamStruct,
                          inout outputStruct: SMCParamStruct) -> kern_return_t {
+        return callSMC(&inputStruct, outputStruct: &outputStruct, selector: Selector.kSMCHandleYPCEvent.rawValue)
+    }
+    
+    /**
+    Make a call to the SMC
+    
+    - parameter inputStruct: Struct that holds data telling the SMC what you want
+    - parameter outputStruct: Struct holding the SMC's response
+    - returns: IOKit return code
+    */
+    private func callSMC(inout inputStruct : SMCParamStruct,
+        inout outputStruct: SMCParamStruct, selector: UInt32) -> kern_return_t {
```

Comment:
Instead of overloading the function, would be cleaner to just set a default value for the `selector` parameter to `kSMCHandleYPCEvent`. See `getTemperature()` method signature for example (`unit` parameter). Though I don't see this being called with a custom selector anyway.